Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -3,1292 +3,54 @@ Copyright (c) 2019 Johan Commelin. All rights reserved.
-  fun φ n => coeff n φ
-
-/-- The natural inclusion from multivariate polynomials into multivariate formal power series.-/
-instance coeToMvPowerSeries : Coe (MvPolynomial σ R) (MvPowerSeries σ R) :=
-  ⟨toMvPowerSeries⟩
-#align mv_polynomial.coe_to_mv_power_series MvPolynomial.coeToMvPowerSeries
-
-theorem coe_def : (φ : MvPowerSeries σ R) = fun n => coeff n φ :=
-  rfl
-#align mv_polynomial.coe_def MvPolynomial.coe_def
-
-@[simp, norm_cast]
-theorem coeff_coe (n : σ →₀ ℕ) : MvPowerSeries.coeff R n ↑φ = coeff n φ :=
-  rfl
-#align mv_polynomial.coeff_coe MvPolynomial.coeff_coe
-
-@[simp, norm_cast]
-theorem coe_monomial (n : σ →₀ ℕ) (a : R) :
-    (monomial n a : MvPowerSeries σ R) = MvPowerSeries.monomial R n a :=
-  MvPowerSeries.ext fun m => by
-    classical
-    rw [coeff_coe, coeff_monomial, MvPowerSeries.coeff_monomial]
-    split_ifs with h₁ h₂ <;> first |rfl|subst m; contradiction
-#align mv_polynomial.coe_monomial MvPolynomial.coe_monomial
-
-@[simp, norm_cast]
-theorem coe_zero : ((0 : MvPolynomial σ R) : MvPowerSeries σ R) = 0 :=
-  rfl
-#align mv_polynomial.coe_zero MvPolynomial.coe_zero
-
-@[simp, norm_cast]
-theorem coe_one : ((1 : MvPolynomial σ R) : MvPowerSeries σ R) = 1 :=
-    coe_monomial _ _
-#align mv_polynomial.coe_one MvPolynomial.coe_one
-
-@[simp, norm_cast]
-theorem coe_add : ((φ + ψ : MvPolynomial σ R) : MvPowerSeries σ R) = φ + ψ :=
-  rfl
-#align mv_polynomial.coe_add MvPolynomial.coe_add
-
-@[simp, norm_cast]
-theorem coe_mul : ((φ * ψ : MvPolynomial σ R) : MvPowerSeries σ R) = φ * ψ :=
-  MvPowerSeries.ext fun n => by
-    classical
-    simp only [coeff_coe, MvPowerSeries.coeff_mul, coeff_mul]
-#align mv_polynomial.coe_mul MvPolynomial.coe_mul
-
-@[simp, norm_cast]
-theorem coe_C (a : R) : ((C a : MvPolynomial σ R) : MvPowerSeries σ R) = MvPowerSeries.C σ R a :=
-  coe_monomial _ _
-set_option linter.uppercaseLean3 false in
-#align mv_polynomial.coe_C MvPolynomial.coe_C
-
-set_option linter.deprecated false in
-@[simp, norm_cast]
-theorem coe_bit0 :
-    ((bit0 φ : MvPolynomial σ R) : MvPowerSeries σ R) = bit0 (φ : MvPowerSeries σ R) :=
-  coe_add _ _
-#align mv_polynomial.coe_bit0 MvPolynomial.coe_bit0
-
-set_option linter.deprecated false in
-@[simp, norm_cast]
-theorem coe_bit1 :
-    ((bit1 φ : MvPolynomial σ R) : MvPowerSeries σ R) = bit1 (φ : MvPowerSeries σ R) := by
-  rw [bit1, bit1, coe_add, coe_one, coe_bit0]
-#align mv_polynomial.coe_bit1 MvPolynomial.coe_bit1
-
-@[simp, norm_cast]
-theorem coe_X (s : σ) : ((X s : MvPolynomial σ R) : MvPowerSeries σ R) = MvPowerSeries.X s :=
-  coe_monomial _ _
-set_option linter.uppercaseLean3 false in
-#align mv_polynomial.coe_X MvPolynomial.coe_X
-
-variable (σ R)
-
-theorem coe_injective : Function.Injective (Coe.coe : MvPolynomial σ R → MvPowerSeries σ R) :=
-    fun x y h => by
-  ext
-  simp_rw [← coeff_coe]
-  congr
-#align mv_polynomial.coe_injective MvPolynomial.coe_injective
+import Mathlib.RingTheory.MvPowerSeries.Basic
```

Comment:
```suggestion
import Mathlib.RingTheory.MvPowerSeries.Basic
```

Reply:
What do you mean?

Reply:
@faenuccio I guess you did something wrong with the github interface :)

Reply:
Indeed, I am sorry. I am fixing it now in this moment....